Library/Homebrew/cmd/update-report.rb

```diff
@@ -182,7 +182,7 @@ def output_update_report
       end
       if reporter.updated?
         updated_taps << tap.name
-        hub.add(reporter, preinstall: args.preinstall?)
+        hub.add(reporter, filter_null_modifications: !args.preinstall? && !args.quiet?)
```

Comment:
Not huge on the `filter_null_modifications` naming. How about:
- `only_show_modified_formulae_with_version_changes`?
- `only_show_upgraded_modified_formulae`
- `filter_from_formula_versions`

or something else?

Reply:
Yep! I'm good with those. I think I'll run with `only_show_upgraded_modified_formulae` and push a new commit.

Reply:
✅  Change made.